tests/data/accelerator.json

```diff
@@ -1 +1 @@
[]budv+l4-sL":{"opcode":"bender_menu_SPRITE_PROPS","next":null,"parent":"#T+=NNLv/piw1gEX0FyH","inputs":{},"fields":{"SPRITE_PROPS":["height"]},"shadow":true,"topLevel":false},"(V%2k4V/cf^T-k~vHBog":{"opcode":"bender_spriteInfo","next":null,"parent":"BfSUpp2]f*ep82h9R_w1","inputs":{"PROPERTY":[1,";=50+F^u8OhGQRw|w1x3"]},"fields":{},"shadow":false,"topLevel":false},";=50+F^u8OhGQRw|w1x3":{"opcode":"bender_menu_SPRITE_PROPS","next":null,"parent":"(V%2k4V/cf^T-k~vHBog","inputs":{},"fields":{"SPRITE_PROPS":["width"]},"shadow":true,"topLevel":false},"~UM4Jn/YL~+=OW.Riq|U":{"opcode":"bender_spriteInfo","next":null,"parent":"NsSz~f:-jC(YMQTQuX+A","inputs":{"PROPERTY":[1,"s4Dlzfz/W4sK%gl=@;Zk"]},"fields":{},"shadow":false,"topLevel":false},"s4Dlzfz/W4sK%gl=@;Zk":{"opcode":"bender_menu_SPRITE_PROPS","next":null,"parent":"~UM4Jn/YL~+=OW.Riq|U","inputs":{},"fields":{"SPRITE_PROPS":["width"]},"shadow":true,"topLevel":false},"V?Y,KQxnp#3;[[RVC-]v":{"opcode":"bender_spriteInfo","next":null,"parent":"td:{(1{0gbx^53.0]azK","inputs":{"PROPERTY":[1,"A*M;R}(I*1X%7s+!$Ps9"]},"fields":{},"shadow":false,"topLevel":false},"A*M;R}(I*1X%7s+!$Ps9":{"opcode":"bender_menu_SPRITE_PROPS","next":null,"parent":"V?Y,KQxnp#3;[[RVC-]v","inputs":{},"fields":{"SPRITE_PROPS":["height"]},"shadow":true,"topLevel":false},"6kGEQ1{gc4C)]ABh4i=:":{"opcode":"bender_spriteInfo","next":null,"parent":"9]y{XCb`WvYgdGZUTVOV","inputs":{"PROPERTY":[1,"n`NP!M5u?B7N`47Pw3Iu"]},"fields":{},"shadow":false,"topLevel":false},"n`NP!M5u?B7N`47Pw3Iu":{"opcode":"bender_menu_SPRITE_PROPS","next":null,"parent":"6kGEQ1{gc4C)]ABh4i=:","inputs":{},"fields":{"SPRITE_PROPS":["width"]},"shadow":true,"topLevel":false},"s4|98:|`0Dhx_j[[SQ}j":{"opcode":"bender_spriteInfo","next":null,"parent":"S8PS.]~=@:}[]eT=3#4!","inputs":{"PROPERTY":[1,"1O`2`mK]4/Z:mX3ajBOv"]},"fields":{},"shadow":false,"topLevel":false},"1O`2`mK]4/Z:mX3ajBOv":{"opcode":"bender_menu_SPRITE_PROPS","next":null,"parent":"s4|98:|`0Dhx_j[[SQ}j","inputs":{},"fields":{"SPRITE_PROPS":["height"]},"shadow":true,"topLevel":false},"QN*.A+x3[7,8DWPR9b(;":{"opcode":"bender_spriteInfo","next":null,"parent":"V)t^q;Nc.aN18ydH:1lN","inputs":{"PROPERTY":[1,"I`Xc:k~Mn{h5!RsgAZd_"]},"fields":{},"shadow":false,"topLevel":false},"I`Xc:k~Mn{h5!RsgAZd_":{"opcode":"bender_menu_SPRITE_PROPS","next":null,"parent":"QN*.A+x3[7,8DWPR9b(;","inputs":{},"fields":{"SPRITE_PROPS":["width"]},"shadow":true,"topLevel":false},"cs+q_{.ZKcfox.:8KOZQ":{"opcode":"bender_spriteInfo","next":null,"parent":"EMc{4tDH?a49qb{m9ypU","inputs":{"PROPERTY":[1,"B,#-;Rx;_|u#NV!aI6r-"]},"fields":{},"shadow":false,"topLevel":false},"B,#-;Rx;_|u#NV!aI6r-":{"opcode":"bender_menu_SPRITE_PROPS","next":null,"parent":"cs+q_{.ZKcfox.:8KOZQ","inputs":{},"fields":{"SPRITE_PROPS":["height"]},"shadow":true,"topLevel":false},"E9ZQl(k=:[^Hk~p93!3d":{"opcode":"bender_spriteInfo","next":null,"parent":"2/0Nfcp|Uxb`k1~o_VHW","inputs":{"PROPERTY":[1,"Bnw)okD~mrz|,?Kw7JhT"]},"fields":{},"shadow":false,"topLevel":false},"Bnw)
```

Comment:
can you just reset this change (`git checkout master tests/data/accelerator.json` and commit it?)